PURL-SPECIFICATION.rst

```diff
@@ -259,14 +257,31 @@ These ``purl`` separator characters MUST NOT be percent-encoded when used as
 Percent-encoding rules
 ----------------------
 
-When applying percent-encoding or decoding to a string, use the rules of RFC
-3986 section 2 (https://datatracker.ietf.org/doc/html/rfc3986#section-2).
+Unless otherwise provided in this specification, when applying percent-encoding
+or decoding to a string, use the rules of RFC 3986 section 2
+(https://datatracker.ietf.org/doc/html/rfc3986#section-2).  In the event of any
+conflict between this specification and RFC 3986 section 2, this specification
+governs.
+
+In the "Rules for each ``purl`` component" section above, each component
+defines when and how to apply percent-encoding and decoding to its content.
+
+When percent-encoding is required, all Permitted Characters MUST be encoded as
+UTF-8 and then percent-encoded except for the following:
+
+- the alphanumeric characters,
```

Comment:
This needs to specify which alphanumeric characters. Most people will probably understand that ß does not count, but it is a Unicode letter.

Reply:
@mjherzog I think that Matts='s point may be that we re-spell out other characters below but not here. It does not hurt to list things here again IMHO, does it?
Something like:
 ```suggestion
- the alphanumeric characters ``A to Z``, ``a to z``, ``0 to 9``,
```

Matt, was this your point?